src/e2e/step-definitions/common.ts

```diff
@@ -68,7 +68,7 @@ Given(
   { timeout: ONE_MINUTE_IN_MS },
   async function (this: CustomWorld) {
     await this.page.waitForResponse(/zoeken\/list/);
-    await this.page.waitForTimeout(100);
+    await this.page.waitForTimeout(1000);
```

Comment:
Hard to see what the unit is here. I guess milliseconds? Adding this as a constant does it make it more readable I think.
```suggestion
    await this.page.waitForTimeout(ONE_SECOND_IN_MS);
```

With 

```const ONE_SECOND_IN_MS = 1_000;
```
on top of this file (cannot seem to make a suggestion there..)